guides/doc-Upgrading_and_Updating/topics/upgrading_process_overview.adoc

```diff
@@ -61,10 +48,26 @@ For more information, see the Red Hat Knowledgebase solution https://access.redh
 [[upgrade_paths]]
 == Upgrade Paths
 
+ifdef::satellite[]
+Before you upgrade your {Project}, you have to upgrade the operating system on your {ProjectServer}s and {SmartProxies} to {RHEL} 8.
+endif::[]
+ifndef::satellite[]
+Before you upgrade your {Project}, you have to upgrade the operating system on your {ProjectServer}s and {SmartProxies} to Enterprise Linux 8.
+endif::[]
+
+There are two ways of upgrading your OS:
+
+* xref:upgrading-project-in-place-using-leapp_{context}[]
+* xref:migrating-project-to-el8_{context}[]
+
+=== Upgrading {ProjectName}
+
 ifdef::satellite[]
 You can upgrade to {ProjectName} {ProjectVersion} from {ProjectName} {ProjectVersionPrevious}.
 {ProjectServer}s and {SmartProxyServer}s on earlier versions must first be upgraded to {Project} {ProjectVersionPrevious}.
-For more information, see the {Project} {ProjectVersionPrevious} https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html/upgrading_and_updating_red_hat_satellite/[Upgrading and Updating {ProjectName}] guide.
+For more information, see the https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html/upgrading_and_updating_red_hat_satellite/[_{UpgradingDocTitle} to {ProjectVersionPrevious}_].
```

Comment:
Can this link to `{SomethingDocURL}`?

Reply:
We would have to define DocURLs for pervious versions and it's for a greater debate whether that much work is actually benefical. If you want to consider it, please file an issue.